lib/puppet_x/puppetlabs/cd4pe_client.rb

```diff
@@ -9,12 +9,12 @@ module PuppetX::Puppetlabs
   class CD4PEClient < Object
     attr_reader :config
 
-    LOGIN_ENDPOINT = '/login'.freeze
-    ROOT_AJAX_ENDPOINT = '/root/ajax'.freeze
-    ROOT_ENDPOINT_SETTINGS = '/root/endpoint-settings'.freeze
-    ROOT_STORAGE_SETTINGS = '/root/storage-settings'.freeze
-    SIGNUP_ENDPOINT = '/signup'.freeze
-    HW_CONFIG_ENDPOINT = '/root/hw-config'.freeze
+    LOGIN_ENDPOINT = '/cd4pe/login'.freeze
```

Comment:
Either this is a breaking change that you use with CD4PE 4.5.0+, or we need to make it smarter or more configurable.